lang/py/src/avro/io.py

```diff
@@ -280,6 +296,64 @@ def read_utf8(self):
     """
     return unicode(self.read_bytes(), "utf-8")
 
+  def read_date_from_int(self):
+    """
+    int is decoded as python date object
```

Comment:
nitpick: can we add some punctuations between sentences so that it's easier to read?